### ISSUE TITLE
Add contribution guide

### ISSUE THREAD

**Issue by jnovick** [ppumkin/PrestoCoverage#15]
Hey, I started creating a very similar extension, but before pushing to a git repository found your repo by looking at this issue: https://github.com/tonerdo/coverlet/issues/32. I figured instead of creating a competing project, I could contribute to yours. Unfortunately, I am having trouble getting your project to compile for me because of the coverlet.console package. I am looking for advice on contributing and how to get started. Should I be waiting until https://github.com/tonerdo/coverlet/issues/212 is resolved to contribute that way it is published publicly. I tried publishing it myself but had an issue since it is .net core not .net standard. Let me know how to get started with contributions. Thanks.

**Comment by ppumkin** [comment]
Oh Hi. Yes sorry for late reply. I was busy with with real life.
That package is an "unofficial" package I made from the real source code. I have contacted the guy from Coverlet if I can publish it but he does not reply. (currently I am hosting it privately)

I will take a look and let you know. 

I am happy you would like to contribute. I hope its not too late. Please let me sort out a open PR I am not sure what I did last time and need to catch up

**Comment by ppumkin** [comment]
Here is the NuGet Package- You can load it via using file sytem in NuGet config

[coverlet.core.2018.10.10.2.nupkg.zip](https://github.com/ppumkin/PrestoCoverage/files/3005740/coverlet.core.2018.10.10.2.nupkg.zip)

**Comment by ppumkin** [comment]
I have added a public NuGet for Coverlet.Core that I will be using.
It is on my own decision to publish it.. if tonerdo changes his mind and does his own version I will pull this package and update accordingly

**Comment by jnovick** [comment]
Thanks, I am busy this week but I am gonna take a look this weekend and possibly put up a couple MRs if I can get it building now that the package is available.

**Comment by ppumkin** [comment]
I am in process of trying to update my source to use that package. will update master once I get it compiling

**Comment by ppumkin** [comment]
I made a forked version of Coverlet.Core and published it under [piotrkula.coverlet.core](https://www.nuget.org/packages/piotrkula.coverlet.core/) but I will update the reference in the project so this should not be a problem any more.

Also I forked the latest version of Coverlet.Core which has a few extra parameters it want on the constructor so it wont build straight away. This plugin will probably not benefit from any of those things and is just a line fix to get going again. Will PR that soon

I went to a machine I never used for developing PrestoCoverage and documented what I needed to do to get it running and made this quick guide. Should get you going.

https://github.com/ppumkin/PrestoCoverage/wiki
